dist/plugins/TestPluginWithJarFile/resources/configurationFiles/analysisTest.xml

```diff
@@ -21,11 +21,11 @@
         <cvalue name="testInnerOutputDirectory" value="${testOutputDirectory}/testfilesw2"/>
     </configurationvalues>
     <processingTools>
-        <tool name="cleanupScript" value="cleanupScript.sh" basepath="roddyTests">
-            <resourcesets>
-                <rset size="l" memory="0.1" cores="1" nodes="1" walltime="1"/>
-            </resourcesets>
-        </tool>
+        <!--<tool name="cleanupScript" value="cleanupScript.sh" basepath="roddyTests">-->
```

Comment:
No, noone ever used it. However it is available as an option. In this case, it is also not there and Roddy would fail.